lib/ember-cli/html_page.rb

```diff
@@ -3,9 +3,36 @@ class HtmlPage
     def initialize(options)
       @content = options.fetch(:content)
       @asset_resolver = options.fetch(:asset_resolver)
+      @head = options.fetch(:head, "")
     end
 
     def render
+      if has_head_tag?
+        insert_head_content
+      end
+
+      html
+    end
+
+    private
+
+    def insert_head_content
+      html.insert(head_tag_index, @head.to_s)
+    end
+
+    def html
+      @html ||= resolved_html
+    end
+
+    def head_tag_index
+      html.index("</head") || -1
+    end
+
+    def has_head_tag?
```

Comment:
Minor: what do you think about making this the first private method? I think it's nice to see the methods in the order that you'll read while going down the file.